json_serializable/test/src/json_serializable_test_input.dart

```diff
@@ -104,6 +104,21 @@ class UnknownFieldType {
   Bob number;
 }
 
+@JsonSerializable(createFactory: false)
+class UnknownFieldTypeToJsonOnly {
+  // ignore: undefined_class
+  Bob number;
```

Comment:
Note that in Dart 2 this will be a compile time error so migrating this test to use modular kernel will likely be an issue, but you could cross that bridge when you come to it (or possibly exclude this file from that builder).

Reply:
ack – it was either this or depend on Flutter in the Test 😄